Library/Homebrew/rubocops/homepage.rb

```diff
@@ -58,6 +58,14 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
           when %r{^http://([^/]*)\.(sf|sourceforge)\.net(/|$)}
             problem "#{homepage} should be `https://#{Regexp.last_match(1)}.sourceforge.io/`"
 
+          when /readthedocs\.org/
+            offending_node(parameters(homepage_node).first)
+            problem "#{homepage} should be `#{homepage.sub("readthedocs.org", "readthedocs.io")}`"
+
+          when %r{^https://github.com.*\.git}
+            offending_node(parameters(homepage_node).first)
+            problem "GitHub URLs (`#{homepage}`) should not end with .git"
```

Comment:
Maybe `GitHub homepages` instead of `GitHub URLs`? This a little bit sounds like _all_ GitHub URLs should not end with `.git`

Reply:
Yeah I can see how that could be confusing. I'll update it.